docs.yaml

```diff
@@ -18,6 +18,11 @@ sections:
     sources:
       - type:  markdown
         files: 'upgrade_guide/**/*.md'
+  - title:     Reference configuration
+    prefix:    /reference_conf
+    sources:
+      - type:  rst
+        files: 'reference_conf/**/*.md'
```

Comment:
For some reason this doesn't work if the file uses the extension `rst`, I'll look into it further tomorrow.